README.md

```diff
@@ -127,16 +127,15 @@ Mining, hashrates, `ssh,` and `db` methods are not supported.
 * `eth_estimateGas` will always return `0x0`. Near RPC does not seem to support
   gas estimation for calls.
 
-* Many fields in Ethereum data structures do not have a direct correspondance
-  to Near data structures. Some attributes of transaction and block objects
-  will be unreliable.
+* Some fields in Ethereum data structures do not have a direct correspondance to Near data structures, therefore these fields are unimplemented or unreliable. Always consult implementation in near_to_eth_objects.js. Examples include:
+  * The tx receipts `status` field is always `0x1`, regardless of the success of the transaction.
+  * `transactionIndex` is constant, and not reliable.
 
-* Some fields are just unimplemented. For example, tx receipts currently always
-  have a `0x1` status, regardless of the success of the transaction.
+* Some calls, like `eth_getTransactionByBlockHashAndIndex` have no sensible correspondance to Near block structure.
 
-* Some fields are just unimplemented. Always consult implementation in `near_to_eth_objects.js`. E.g. tx index is constant, and not reliable.
-
-* Some calls, like `eth_getTransactionByBlockHashAndIndex` have no sensible correspondance to Near block structure
+* Near doesn't keep track of nonces for contracts which is a required feature for the evm. Therefore both an evm nonce and near nonce exist for each account. `eth_getTransactionCount` will return a reference to the evm nonce. All other nonce fields in the web3-near-provider reference the near nonce.
+    * evm nonce: number of evm transactions executed by account/address
```

Comment:
normally this would be correct. in this case it might be better as `the nonce used for address calculation in the next  EVM contract deployment from this EVM account.`